charts/argo-rollouts/templates/controller/deployment.yaml

```diff
@@ -31,6 +31,9 @@ spec:
         {{- range $key, $value := . }}
         {{ $key }}: {{ $value | quote }}
         {{- end }}
+        {{/* Add the checksum of the ConfigMap to the annotations */}}
+        {{/* This ensures that ConfigMap changes trigger a rolling update */}}
+        configmap-checksum: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
We want to align to other argoproj's checksum, and I don't think we need comments for this. :)
e.g.) https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/templates/argocd-application-controller/deployment.yaml#L24